.github/workflows/ci.yml

```
@@ -6,11 +6,19 @@ on:
   workflow_dispatch: {}
   workflow_call: {}
   push:
+    tags-ignore:
+      - 'v99*'
+    paths-ignore:
+      - 'test/**'
   pull_request:
+    tags-ignore:
```

actionlint (1.7.7)

"tags-ignore" filter is not available for pull_request event. it is only for push event

(events)